File: codeforces/76.maximum_gcd_[1370A]/maximum_gcd.py
# Author: José Rodolfo (jric2002)
t = int(input())
while (t):
  n = int(input())
  max_value = 0
  if ((n % 2) == 0):
    max_value = int(n / 2)
  else:
    max_value = int((n - 1) / 2)
  print(max_value)
  t -= 1

File: codeforces/76.maximum_gcd_[1370A]/maximum_gcd.cpp
#include <iostream>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int t;
  unsigned int n, max_value;
  cin >> t;
  while (t--) {
    cin >> n;
    if ((n % 2) == 0) {
      max_value = n / 2;
    }
    else {
      max_value = (n - 1) / 2;
    }
    cout << max_value << endl;
  }
  return 0;
}